doc/conf.py

```diff
@@ -22,17 +22,22 @@
     "pytools": ("https://documen.tician.de/pytools", None),
 }
 
-nitpick_ignore_regex = [
-    ["py:class", r"numpy._?typing._generic_alias.ScalarType"],
-]
```

Comment:
Not sure why, but this error doesn't seem to be triggering for me locally (maybe an older numpy version thing?)